pages/common/git-clone.md

```diff
@@ -15,13 +15,9 @@
 
 `git clone --no-checkout {{remote_repository_location}}`
 
-- Clone a local repository:
+- Clone a local repository quietly:
 
-`git clone --local {{path/to/local/repository}}`
-
-- Clone quietly:
-
-`git clone --quiet {{remote_repository_location}}`
+`git clone --local --quiet {{path/to/local/repository}}`
```

Comment:
We only allow at most 8 examples per page, how about merging these 2 examples?

Reply:
yes makes sense